<a href="https://colab.research.google.com/github/gurram46/Chi-Square-test/blob/main/Hypothesis_Testing.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Hypothesis Testing


Background:

Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.

Objective:

To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.

Data Provided:

The theoretical weekly operating cost model: W = $1,000 + $5X
Sample of 25 restaurants with a mean weekly cost of Rs. 3,050
Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units


# Assignment Tasks:


**State the Hypotheses statement :**



The goal is to test if the actual weekly operating costs exceed the cost model. The hypotheses are:

Null Hypothesis (𝐻0): The actual mean weekly operating cost is equal to or less than the theoretical cost given by the model.

Alternative Hypothesis (𝐻1): The actual mean weekly operating cost is higher than the theoretical cost.

SO,

H0: mue(u)=< theoretical cost

H1: mue(u)> theoretical cost


 Calculate the Test Statistic:

Use the following formula to calculate the test statistic (t):

where:

ˉxˉ = sample mean weekly cost (Rs. 3,050)

μ = theoretical mean weekly cost according to the cost model (W = $1,000 + $5X for X = 600 units)

σ = 5*25 units

n = sample size (25 restaurants)


In [6]:
import pandas as pd
import numpy as np
import scipy.stats as stats

In [2]:
sample_mean = 3050
theoretical_mean = 1000+5*600
sample_size=25
standard_deviation=5*25

In [4]:
print(sample_mean)
print(theoretical_mean)
print(sample_size)
print(standard_deviation)

3050
4000
25
125


So,

sample mean = 3050

theoretical mean = 4000

sample mean = 25

standard deviation = 125

In [8]:
std_error =standard_deviation/np.sqrt(sample_size)

In [9]:
std_error

25.0

In [10]:
t_value=(sample_mean-theoretical_mean)/std_error

In [11]:
t_value

-38.0

**Determine the Critical Value:**

Using the alpha level of 5% (α = 0.05), determine the critical value from the standard normal (Z) distribution table.

In [14]:
alpha = 0.05

In [15]:
from scipy.stats import norm
critical_value = norm.ppf(1-alpha)

In [16]:
critical_value

1.6448536269514722

We Accept null hypothesis because t-value is lesser then critical-value

so Ho>H1

**Make a Decision:**

Compare the test statistic with the critical value to decide whether to reject the null hypothesis.


In [17]:
if t_value > critical_value:
    print("Reject the null hypothesis. The actual mean weekly operating cost is higher than the theoretical cost.")
else:
    print("Fail to reject the null hypothesis. There is no significant evidence to suggest the actual mean weekly operating cost is higher than the theoretical cost.")

Fail to reject the null hypothesis. There is no significant evidence to suggest the actual mean weekly operating cost is higher than the theoretical cost.


**Conclusion:**

Based on the decision in step 4, conclude whether there is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.


In [18]:
if t_value > critical_value:
  print("There is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.")
else:
  print("There is no strong evidence to support the restaurant owners' claim that the weekly opersting costs are higher then the model suggests")


There is no strong evidence to support the restaurant owners' claim that the weekly opersting costs are higher then the model suggests
